src/app/project-overview/components/project-overview/project-overview.component.html

```diff
@@ -219,6 +219,22 @@
             </div>
         </div>
 
+        <div class="mt-8 w-2/4 grid w-full"
+            [ngClass]="displayGraphsForm.value == DisplayGraphsType.ALL || displayGraphsForm.value == DisplayGraphsType.LABEL_DISTRIBUTION?null:'hidden'">
```

Comment:
is this meant to be visible with the LABEL_DISTRIBUTION if you select it or is another state missing here?

Reply:
no, you're right, i missed that. will add it